controllers/controller.js

```diff
@@ -90,4 +90,23 @@ const {
       .catch(err => {
        next(err);
       })
-  };
\ No newline at end of file
+  };
+
+  exports.patchArticle = (req, res, next) => {
+    console.log('in controller')
+    const { article_id } = req.params;
+    const voteIncrement  = req.body;
+      
+    Promise.all([
```

Comment:
You are using 'RETURNING' in your query, so you should get the patched object back even without selecting the article.

Reply:
As Béla said above, we don't need to select the article. Simply using your changeVotes function will be enough. If it returns nothing, the article does not exist.